cmd/api/main.go

```diff
@@ -87,7 +92,11 @@ func main() {
 
 	dbURI := fmt.Sprintf("host=%s user=%s password=%s dbname=%s port=%s sslmode=%s", configFile.DB_HOST, configFile.DB_USER, configFile.DB_PASSWORD, configFile.DB_NAME, configFile.DB_PORT, configFile.DB_SSLMODE)
 
-	db, err := gorm.Open(postgres.Open(dbURI), &gorm.Config{})
+	gormConf := &gorm.Config{}
+	if logConf.Level.Level() == zap.DebugLevel {
+		gormConf.Logger = logger.Default.LogMode(logger.Silent)
+	}
```

Comment:
I think this does the opposite of what we want. Logging higher than "debug" should silence the gorm logger.